codegen/projections/white_label/lib/white_label/types.rb

```diff
@@ -441,6 +441,59 @@ class EndpointWithHostLabelOperationOutput
       attr_accessor(*MEMBERS)
     end
 
+    class Events < Hearth::Union
+      class SimpleEvent < Events
+        def to_h
+          { simple_event: super(__getobj__) }
+        end
+      end
+
+      class NestedEvent < Events
+        def to_h
+          { nested_event: super(__getobj__) }
+        end
+      end
+
+      class ExplicitPayloadEvent < Events
+        def to_h
+          { explicit_payload_event: super(__getobj__) }
+        end
+      end
+
+      class Unknown < Events
+        def initialize(name:, value:)
+          super({name: name, value: value})
+        end
+
+        def to_h
+          { unknown: super(__getobj__) }
+        end
+
+        def to_s
+          "#<WhiteLabel::Types::Unknown #{__getobj__ || 'nil'}>"
+        end
```

Comment:
Addressed upstream to avoid as many unrelated changes as possible and keep this PR as small as I can: https://github.com/smithy-lang/smithy-ruby/commit/496b075a5fd3fdc370e20a730e086079e5a35c27